tests/deployment/vlan_networks.py

```diff
@@ -133,10 +161,25 @@ def create_networks():
 
             # Verify that we can put nodes on more than one network, with
             # different channels:
+
+            # this is a little clumsy. `get_networks` only returns the node
+            # if it's attached natively. So I must reconnect node[2] to net-1
```

Comment:
Are we actually relying on this behavior anywhere? If so, we should document that more clearly, since it isn't obvious. If not, let's just fix it instead of hacking around it here.

Reply:
yeah, let's fix this within this PR. will do it in the next commit.

Reply:
I was wrong about it. In the previous steps, we do connect tagged networks and those are correctly returned by get_networks. I'll get rid of this workaround.